### DISCUSSION TITLE
My copilot account has not been accepted after five months

### DISCUSSION THREAD

**Discussion by habibi4webdesign** [community/community#9871] [Copilot Conversations] [answered]
Hi guys, I registered for GitHub copilot for more than five months, but it has not been approved yet. What's wrong with that? Please help me with this. What should I do?

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see [#9342 (comment)](https://github.com/github/feedback/discussions/9342#discussioncomment-1902060)
